packages/auth/src/accessToken/routes.ts

```diff
@@ -102,18 +102,26 @@ async function rotateToken(
 ): Promise<void> {
   const { id: managementId } = ctx.params
   const token = (ctx.headers['authorization'] ?? '').replace('GNAP ', '')
-  const result = await deps.accessTokenService.rotate(managementId, token)
-  if (result.success == true) {
-    ctx.status = 200
-    ctx.body = {
-      access_token: {
-        access: result.access.map((a) => accessToBody(a)),
-        value: result.value,
-        manage: deps.config.authServerDomain + `/token/${result.managementId}`,
-        expires_in: result.expiresIn
-      }
-    }
-  } else {
-    ctx.throw(404, { message: result.error.message })
+
+  let newToken
+  try {
+    newToken = await deps.accessTokenService.rotate(managementId, token)
+  } catch (error) {
+    const errorMessage = 'Could not rotate token'
+    deps.logger.error({ error: error && error['message '] }, errorMessage)
+    ctx.throw(400, { message: errorMessage })
+  }
+
+  if (!newToken) {
+    ctx.throw(404, { message: 'Token not found' })
```

Comment:
because we use `Extends<AppContext>` now, we properly have the type on `ctx` and it will know that `ctx.throw` has a `never` return type, and as a result, will flag any code under `ctx.throw` as unreachable